cmd/transcriber/call/transcriber_test.go

```diff
@@ -48,6 +48,16 @@ func setupTranscriberForTest(t *testing.T) *Transcriber {
 	require.NoError(t, err)
 	require.NotNil(t, tr)
 
+	dir, err := os.MkdirTemp("", "data")
+	if err != nil {
+		require.NoError(t, err)
+	}
+	os.Setenv("DATA_DIR", dir)
+	t.Cleanup(func() {
+		os.Unsetenv("DATA_DIR")
+		os.RemoveAll(dir)
+	})
```

Comment:
This is needed both to simplify local testing and also to fix an existing test that was passing purely because of a failure creating the OGG file.